MobileFilter/sections/specific_web.txt

```diff
@@ -1400,6 +1400,7 @@ news-postseven.com##div[style$="text-align:center; margin-bottom: 5px; margin-to
 popcornnews.ru##.horizontal-banner
 autodoc.de##.plus-block
 ||k-img.com/script/fixed_bnr_area.js$domain=s.kakaku.com
+s.kakaku.com##.js-adBnr
```

Comment:
This rule does noting.
Added
`kakaku.com##div[style*="width:728px;height:90px;"]`

Reply:
> `kakaku.com##div[style*="width:728px;height:90px;"]`

Excuse me, I think this rule is not working.

Reply:
I post new pull request. please check #138554

Reply:
Can cofirm, `##.js-adBnr` works and `##div[style*="width:728px;height:90px;"]` not.